.circleci/config.yml

```diff
@@ -247,10 +247,15 @@ jobs:
               yarn ybootstrap
 
               # create versioned docs
-              yarn docs-build
-              mv unreleased _book/
+              # comment out until the next version release
+              # yarn docs-build
+              # mv unreleased _book/
             fi
   
+      - add_ssh_keys:
```

Comment:
out of curiosity, what are these keys for?

Reply:
this is a ssh key that is added to circleci config and allow us to write back to github from the built environment: https://circleci.com/docs/2.0/add-ssh-key/